src/PKSim.Infrastructure/ProjectConverter/v7_3/Converter721To730.cs

```diff
@@ -141,7 +141,7 @@ private void convertEvent(PKSimEvent pkSimEvent)
 
       private void updateIsInputStateByNameAndValue(IContainer containerToUpdate, IContainer templateContainer)
       {
-         foreach (var templateParameter in templateContainer.AllParameters(x => x.Visible && x.Editable && x.ValueIsDefined()))
+         foreach (var templateParameter in templateContainer.AllParameters(x => x.Visible && x.Editable && x.ValueIsComputable()))
```

Comment:
That's the difference here. Template paraemter values such as Lipo, fu etc.. are NAN. And so values was not updated 
But we had tests right? Well yes, those were coincindently updated before because the alternative where named Measurement in template and project and as such, parameter were found by path even if they were NAN.

Now I don't care about NaN. I just want to make sure this is not one of this formula that we have internally that cannot be computed

Reply:
I have also updated the test project to use different alternative name so that the I could get a red tests and then green again